metrics/config.go

```diff
@@ -105,6 +107,10 @@ type metricsConfig struct {
 	// format. It defaults to 9090.
 	prometheusPort int
 
+	// prometheusHost is the host where the metrics are exposed in Prometheus
+	// format. It defaults to "0.0.0.0"
```

Comment:
If I understand the logic correctly, you actually mean "an empty host will default to 0.0.0.0 by prometheus"

Reply:
Yes that is the http convention re-used.

Reply:
Thanks for clarifying. I was confused by the comment, thinking that you want to force it to 0.0.0.0 if not set in env.